frontend/src/container/NewDashboard/DashboardVariablesSelection/VariableItem.tsx

```diff
@@ -88,18 +89,26 @@ function VariableItem({
 		(state) => state.globalTime,
 	);
 
+	// logic to detect if its a rerender or a new render/mount
+	const isMounted = useRef(false);
```

Comment:
why is `onMount` handling different ? `onMount` should be handled  via the root triggers ?

Reply:
removed and added root trigger handling